drivers/windows/labels.go

```diff
@@ -30,4 +30,10 @@ const (
 
 	// SourceMac of the network
 	SourceMac = "com.docker.network.windowsshim.sourcemac"
+
+	// EnableICC label
+	EnableICC = "com.docker.network.bridge.enable_icc"
+
+	// DisableDNS label
+	DisableDNS = "com.docker.network.windowsshim.disable_dns"
```

Comment:
Till we have the support for multiple nat networks it looks ok to use a driver specific option to disable DNS on certain endpoints. Please add a comment that this can be removed and the ep's `anonymous` flag can be used when we have support for multiple nat networks.